common/models/user.js

```diff
@@ -301,6 +301,21 @@ module.exports = function(User) {
     return fn.promise;
   };
 
+  User.observe('before delete', function(ctx, next) {
+    var AccessToken = ctx.Model.relations.accessTokens.modelTo;
+    ctx.Model.find({ where: ctx.where }, function(err, list) {
```

Comment:
@raymondfeng 
do you mean: `ctx.Model.find({ where: ctx.where[pkName] }, function(err, list) {`?
this one fails the deleteById test case. 

and if I do it like that: 
`ctx.Model.find({ where: {fields:[pkName]} }, function(err, list) {`
it fails both and doesn't delete the token
Am I missing something here?